Parser/Bonding/Parser.ts

```diff
@@ -21,30 +20,9 @@ const parseArguments = (args: string) => {
     return [token, type, amount]
 }
 
-function timeConverter(UNIX_timestamp) {
-    const a = new Date(UNIX_timestamp * 1000);
-    const months = ['Jan', 'Feb', 'Mar', 'Apr', 'May', 'Jun', 'Jul', 'Aug', 'Sep', 'Oct', 'Nov', 'Dec'];
-    const year = a.getFullYear();
-    const month = months[a.getMonth()];
-    const date = a.getDate();
-    const hour = a.getHours();
-    const min = a.getMinutes();
-    const sec = a.getSeconds();
-    const time = `${date} ${month} ${year} ${hour}:${min}:${sec}`;
-    return time;
-}
-
 function validateArgs([token, type]: string[]) {
-    const tokens = Object.keys(BondTokens)
-    if (!(tokens.includes(token))) {
-        throw new Error("Incorrect token name " + token + " avalable: " + tokens.toString() )
-    }
-    const types = Object.keys(BondTypes)
-    if (!(types.includes(type))) {
-        throw new Error("Incorrect bond type " + type + " avalable: " + types.toString())
-    }
-    if (token === BondTokens.usdc) {
-        throw new Error("Only FTM bonding is available for now, USDC coming soon")
+    if (!(Object.keys(BondTokens).includes(token)) || !(Object.keys(BondTypes).includes(type))) {
```

Comment:
I'm actually not sure we should delete the more verbose version of error message, since it's still on frontend and wouldn't hurt the user & us while debugging